src/apis/oracle.ts

```diff
@@ -1,15 +1,14 @@
 import { ErrorCode } from "../interfaces/default";
 import { ApiPromise } from "@polkadot/api";
 import { BTreeSet } from "@polkadot/types/codec";
-import { AccountId } from "@polkadot/types/interfaces";
 import { Moment } from "@polkadot/types/interfaces/runtime";
 import { u128 } from "@polkadot/types/primitive";
-import { BTC_IN_SAT, DOT_IN_PLANCK, sendLoggedTx } from "../utils";
+import { sendLoggedTx } from "../utils";
 import Big from "big.js";
 import { AddressOrPair } from "@polkadot/api/types";
 
 const defaultFeedName = "DOT/BTC";
-const granularity = 5;
+const granularity = 2;
```

Comment:
Granularity in the parachain is 5. Why 2?

EDIT: Ok, I see you tried to simplify the conversion. I would keep as is. It is possible that the currencies we are dealing with would change their denomation. Unlikely, but it already happened once for Polkadot so it makes it somewhat cleaner to have it that way. We could move the granularity to a constant into the `constants` file though.

Reply:
The previous granularity was broken and the Oracle test was being skipped. If you recall, the getExchangeRate function returns 100 or 1000 after the rate is set to 1. The current implementation might be an oversimplification, but the previous one was wrong

Reply:
Right, but then let's fix the right things. The granularity on the parachain is 5. Somehow we should reach the correct exchange rate without having inconsistencies between the granularities on the lib and the chain.